src/writeData/clients/Python/initialize.example

```diff
@@ -4,7 +4,7 @@ from influxdb_client import InfluxDBClient, Point, WritePrecision
 from influxdb_client.client.write_api import SYNCHRONOUS
 
 # You can generate an API token from the "API Tokens Tab" in the UI
-token = "<%= token %>"
+token = "<INFLUX_TOKEN>"
```

Comment:
```suggestion
token = os.getenv("INFLUX_TOKEN")
```

Reply:
It is also needed to add an import at the start of example:

```python
import os
```

Reply:
done!